crates/pipeline_manager/src/db/storage.rs

```diff
@@ -396,4 +400,18 @@ pub(crate) trait Storage {
         tenant_name: String,
         provider: String,
     ) -> Result<TenantId, DBError>;
+
+    /// Record information about a compiler binary
+    async fn create_compiled_binary_ref(
+        &self,
+        program_id: ProgramId,
+        version: Version,
+        url: String,
```

Comment:
Maybe give some examples of what URL would look like here .. e.g., wondering if it's file:///path for a local runner or smth else